share/spice/go_watch_it/go_watch_it.js

```diff
@@ -189,7 +189,12 @@
                     item.buy_line = "Available for Purchase";
                     item.rent_line = "";
                 }
-                
+
+                // Fandango has its own suggested line
+                if(item.provider_name === "Fandango") {
+                    item.buy_line = item.suggested_line;
```

Comment:
The suggested line works just fine... I don't know if we should hardcode it instead :8ball: